client/my-sites/earn/memberships/products-list.tsx

```diff
@@ -85,6 +85,11 @@ function ProductsList() {
 		);
 	}
 
+	function onAddNewPaymentPlanButtonClick() {
+		dispatch( recordTracksEvent( 'calypso_memberships_add_payment_plan_click' ) );
```

Comment:
I have seen ` recordTracksEvent(` calls without "dispatch", Is "dispatching" here the recommended Calypso way?

Reply:
Not sure, it was just a copy&paste from:
https://github.com/Automattic/wp-calypso/blob/2616b8da2c63ca8b63582b9a3a7424d4429aecb5/client/my-sites/earn/memberships/section.tsx#L96